tests/testCases.ts

```diff
@@ -1,8 +1,12 @@
-import { ArgumentType } from "../src/light-classnames";
+import {ArgumentType} from "../src";
 
-type testCaseType = { name: string, value: ArgumentType[], expectedResult: string };
+export type TestCase = {
```

Comment:
Maybe,
I think there is no need to write in a name that it is type, just use upper-case.
But because the `Argument` is so general it's worth adding something.

Reply:
merge it into v1 and I'll change it myself. I still think it should be `Argument` because the code itself should be sufficient enough for other programmers to understand that `Argument` is a type